.github/workflows/tests.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
     - name: Setup operator test environment
       uses: charmed-kubernetes/actions-operator@main
       with:
+        charm-channel: 3.x/stable
```

Comment:
How does this help reactive builds?  Isn't this the config that controls the snap version installed by the charm (from [actions.yaml](https://github.com/charmed-kubernetes/actions-operator/blob/main/action.yaml#L31-L34))?

`3.x/stable` seems like a weird value here.

Reply:
ha!  sorry, it's the [charm snap](https://snapcraft.io/charm) channel.  derp.  carry on.